src/Test/Utilities/Portable/Traits/Traits.cs

```diff
@@ -15,6 +15,7 @@ public static class Editors
         public const string Feature = nameof(Feature);
         public static class Features
         {
+            public const string AdditionalInternalStructureOutlinings = nameof(AdditionalInternalStructureOutlinings);
```

Comment:
why do we need this trait?

Reply:
Help find those which provide internal structure outlinings.

Reply:
Please don't add this added trait. It's far too granular to be all that useful.

Reply:
It was useful for me, during developement to find then with the unit test explorer window.